src/applications/hca/components/FormAlerts/SubmissionErrorAlert.jsx

```diff
@@ -9,45 +9,67 @@ const SubmissionErrorAlert = () => {
 
   return (
     <div className="hca-error-message vads-u-margin-bottom--4">
-      <va-alert status="error" uswds>
+      <va-alert status="error">
         <h3 slot="headline">We didn’t receive your online application</h3>
-        <p>
-          We’re sorry. Something went wrong when you tried to submit your
-          application. Try again later.
-        </p>
+        <div>
+          <p className="vads-u-margin-top--0">
+            We’re sorry. Something went wrong when you tried to submit your
+            application. If you were signed in, you can try to submit your
+            application again later. If you were not signed in, you may need to
+            fill out and submit the online application again.
+          </p>
 
-        <p>
-          <strong>
-            If you’re trying to apply by the September 30th special enrollment
-            deadline for certain combat Veterans
-          </strong>
-          , you can also apply in other ways:
-        </p>
-        <ul>
-          <li>
-            Call us at <va-telephone contact={CONTACTS['222_VETS']} />. We’re
-            here Friday, September 29, 7:00 a.m. to 9:00 p.m.{' '}
+          <h4 className="vads-u-font-size--h5">Other ways to apply</h4>
+          <ul>
+            <li>
+              You can call our toll-free hotline at{' '}
+              <va-telephone contact={CONTACTS['222_VETS']} /> to apply by phone,
+              Monday through Friday, 8:00 a.m. to 8:00 p.m.{' '}
+              <dfn>
+                <abbr title="Eastern Time">ET</abbr>
+              </dfn>
+              , <strong>or</strong>
+            </li>
+            <li>
+              You can download and fill out the application. Send your completed
+              application here:
+            </li>
+          </ul>
+          <p className="va-address-block vads-u-margin-bottom--2 vads-u-margin-x--0">
+            Health Eligibility Center <br />
+            2957 Clairmont Road NE, Ste 200 <br />
+            Atlanta, GA 30329-1647 <br />
+          </p>
+          <p>
+            If you have trouble downloading your application, call us at{' '}
+            <va-telephone contact={CONTACTS.HELP_DESK} /> (
+            <va-telephone contact={CONTACTS['711']} tty />
+            ). We’re here Monday through Friday, 8:00 a.m. to 8:00 p.m.{' '}
             <dfn>
-              <abbr title="Central Time">CT</abbr>
+              <abbr title="Eastern Time">ET</abbr>
             </dfn>
-            , and Saturday, September 30, 7:00 a.m. to 11:59 p.m.{' '}
+            .
+          </p>
+
+          <a
```

Comment:
Why not use a `va-link` for this?

Reply:
`va-link` doesnt allow for targets other than "self". ☹️